web/src/components/FilingsCalendar.tsx

```diff
@@ -93,18 +105,28 @@ export const FilingsCalendar = (props: Props): ReactElement => {
 
     return (
       <div data-testid="filings-calendar">
-        <div className="flex flex-align-end padding-top-2">
-          <h2 className="margin-bottom-0">{Config.dashboardDefaults.calendarHeader}</h2>
-          <div className="margin-top-05">
-            <ArrowTooltip title={Config.dashboardDefaults.calendarTooltip}>
-              <div
-                className="fdr fac margin-left-1 margin-bottom-05 font-body-lg text-green"
-                data-testid="calendar-tooltip"
-              >
-                <Icon>help_outline</Icon>
-              </div>
-            </ArrowTooltip>
+        <div className="flex flex-align-end padding-top-2 flex-justify">
+          <div className="flex flex-align-end">
+            <h2 className="margin-bottom-0">{Config.dashboardDefaults.calendarHeader}</h2>
+            <div className="margin-top-05">
+              <ArrowTooltip title={Config.dashboardDefaults.calendarTooltip}>
+                <div
+                  className="fdr fac margin-left-1 margin-bottom-05 font-body-lg text-green"
+                  data-testid="calendar-tooltip"
+                >
+                  <Icon>help_outline</Icon>
+                </div>
+              </ArrowTooltip>
+            </div>
           </div>
+          <Button
+            style="light"
+            className="font-body-2xs padding-y-05 padding-x-1"
+            onClick={handleCalendarOnclick}
+          >
+            <Icon className="usa-icon--size-3 margin-right-05">list</Icon>
+            {Config.dashboardDefaults.calendarListViewButton}
+          </Button>
```

Comment:
I'm not seeing this button styled and left-aligned the way it is in the designs 
![image](https://user-images.githubusercontent.com/6563663/184656276-0f9af296-57c4-47f6-9a2e-3d979e15cd1e.png)

Reply:
I could not duplicate this behavior.